DependencyInjection/TrikoderOAuth2Extension.php

```diff
@@ -64,7 +65,7 @@ private function configureAuthorizationServer(ContainerBuilder $container, array
     {
         $authorizationServer = $container
             ->getDefinition('league.oauth2.server.authorization_server')
-            ->replaceArgument('$privateKey', $config['private_key'])
+            ->replaceArgument('$privateKey', new CryptKey($config['private_key'], null, false))
```

Comment:
We should also accompany the changes in tests as well. Check out the following places:

* `createAuthorizationServer` in `Tests/Integration/AbstractIntegrationTest.php`
* `createResourceServer` in `Tests/Integration/AbstractIntegrationTest.php`
* `generateJwtToken` in `Tests/TestHelper.php`

Reply:
Done